app/main.py

```diff
@@ -1,13 +1,20 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {"errors": [format_linter_error(error) for error in errors],
+            "path": file_path,
+            "status": ["failed" if errors else "passed"][0]}
```

Comment:
do you really need an additional list here as a value?

Reply:
I didn't know how to pass if else without a list =) And now I know that it's real. Thank you!